activate.php

```diff
@@ -125,9 +125,18 @@ public function delete_options() {
 	 * @since 4.1.8 Fixed #425
 	 */
 	public function enable_addons() {
+		// Return if `anspress_installed` plugin option is available.
+		if ( ap_opt( 'anspress_installed' ) ) {
```

Comment:
@mebishalnapit We already have a option key `ap_installed`.

Reply:
@rahularyan Modified to use `ap_installed` option key.